internal/paging/paging.go

```diff
@@ -0,0 +1,23 @@
+package paging
```

Comment:
Could also be `list` package if we add label filtering to the same package 🤔 Easy to rename later as well.

Reply:
Renaming later, if needed, sounds reasonable.